### PR TITLE
[chores, fix] Add pretrain VL right models to zoo, fixes for masked datasets

### PR DESCRIPTION
**Pull request by vedanuj**
- Adds all pretrained models for Pretrain VL project
- Fixes masked coco dataset
- Add few missing configs symlinks
- Fix few configs for train+val

### CODE REVIEW

File: mmf/configs/zoo/models.yaml [hunk 1]
```diff
@@ -1,12 +1,105 @@
 visual_bert:
   pretrained:
     coco:
-      version: 1.0_2020_04_20
-      resources:
-      - url: mmf://models/visual_bert/visual_bert.pretrained.coco.tar.gz
-        file_name: visual_bert.pretrained.coco.tar.gz
-        hashcode: 9af0b8101579d7587d70d7315940310c7fc5ef7269cba497780922e65e4e000d
-    defaults: ${visual_bert.pretrained.coco}
+      full:
```
Comment (apsdehal) [inline mmf/configs/zoo/models.yaml:4]: Add defaults to full for visual_bert.pretrained.coco

File: mmf/configs/zoo/models.yaml [hunk 2]
```diff
@@ -1,12 +1,105 @@
...
+          file_name: visual_bert.pretrained.coco.tar.gz
+          hashcode: 3b4d30768180b375a7d71a74568232a40f780a0774f9c624032081b083793727
+      full_train_val:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.coco_train_val.tar.gz
+          file_name: visual_bert.pretrained.coco_train_val.tar.gz
+          hashcode: b051b49d35923032e26fcd4bfd64688042acabd2a90893ffa90595be8ce05b90
+    vqa2:
+      full:
```
Comment (apsdehal) [inline mmf/configs/zoo/models.yaml:29]: Same here.

File: mmf/configs/zoo/models.yaml [hunk 3]
```diff
@@ -1,12 +1,105 @@
...
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_full_coco_generated.tar.gz
+          file_name: visual_bert.pretrained.cc_full_coco_generated.tar.gz
+          hashcode: 27acc32ab55a9f771d354fec8c2c76fed3464fa0b569dd465b0552c0a79afca5
+      half_coco_generated:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_half_coco_generated.tar.gz
+          file_name: visual_bert.pretrained.cc_half_coco_generated.tar.gz
+          hashcode: a35cdfab35bcb23956bff29f38253aeb4adbec5518ea0a023232f0107f827261
+      small_coco_generated:
```
Comment (apsdehal) [inline mmf/configs/zoo/models.yaml:96]: Can we add a comment here on what exactly is small and half? and link to the paper?
